gocd/templates/pipelines/peanutbutter.libsonnet

```diff
@@ -0,0 +1,53 @@
+// local getsentry = import 'github.com/getsentry/gocd-jsonnet/libs/getsentry.libsonnet';
+local gocdtasks = import 'github.com/getsentry/gocd-jsonnet/libs/gocd-tasks.libsonnet';
+
+function(region) {
+  environment_variables: {
+    SENTRY_REGION: region,
+  },
+  lock_behavior: 'unlockWhenFinished',
+  materials: {
+    peanutbutter_repo: {
+      git: 'git@github.com:getsentry/peanutbutter.git',
+      shallow_clone: true,
+      branch: 'master',
+      destination: 'peanutbutter',
+    },
+  },
+  stages: [
+    {
+      checks: {
+        fetch_materials: true,
+        jobs: {
+          checks: {
+            timeout: 1200,
+            elastic_profile_id: 'peanutbutter',
+            tasks: [
+              gocdtasks.script(importstr '../bash/cloudbuild-checks.sh'),
+            ],
```

Comment:
Symbolicator also has "GitHub checks" task here, see https://github.com/getsentry/symbolicator/blob/master/gocd/templates/bash/github-checks.sh. I assume we don't need that in peanutbutter?

Reply:
I believe it would be good though.

Reply:
I added a check run for the Ubuntu tests.